article/introduction.tex

```diff
@@ -18,3 +18,9 @@ \section{Introduction}
 \\
 Looking at papers such as \cite{Priceaware} it can be seen that propagating the influence of price on users with items as a bridge, as well as integrating category into the propagation progress increases the performance.
 Since \cite{Priceaware} also utilizes a GCN our idea is to improve the performance of the LightGCN framework by utilizing price and category.
+
+We try to extend LightGCN by adding price and categories to the embeddings and adjacency matrix.
+The code for LightGCN and the changes that we made can be seen on \footnote{Github link to LightGCN with our extension: https://github.com/SpecialeBajs/LightGCN}.
+
+Since we got the code for \cite{Priceaware} sent to us personally through email we chose to not make it publicly available.
```

Comment:
```suggestion
Since we got the code for Price-aware Recommendation sent to us personally through email we chose to not make it publicly available as we do not have the licensing or copyright.
```